### PR TITLE
added Python solution to CasingOfString.md

### PR DESCRIPTION
**Pull request by uozcan12**

### CODE REVIEW

File: src/CasingOfString.md [hunk 1]
```diff
@@ -92,3 +93,33 @@ func CheckCase(input string) string {
 	}
 }
 ```
+
+### Python
+```python
+import unicodedata
```
Comment (rahulrk-dev) [inline src/CasingOfString.md:99]: See if you can do it without using any lib so the function can be used by anyone

File: src/CasingOfString.md [hunk 2]
```diff
@@ -92,3 +93,33 @@ func CheckCase(input string) string {
...
+
+def check_case(input_str):
```
Comment (rahulrk-dev) [inline src/CasingOfString.md:101]: function should match with example result above

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]

File: src/CasingOfString.md (hunk 1 above)
Comment (uozcan12) [reply src/CasingOfString.md]: Fixed

File: src/CasingOfString.md (hunk 2 above)
Comment (uozcan12) [reply src/CasingOfString.md]: Fixed

File: src/CasingOfString.md [hunk 3]
```diff
@@ -92,3 +93,32 @@ func CheckCase(input string) string {
 	}
 }
 ```
+
+### Python
+```python
+
+def CheckCase(string):
```
Comment (rahulrk-dev) [inline src/CasingOfString.md:100]: Getting the wrong result mate
image

Comment (rahulrk-dev) [reply src/CasingOfString.md:100]: Spaces are not being handled

**Review by rahulrk-dev — APPROVED** [review]
Thanks @uozcan12, merging

**Pull request merged**